src/imp/exec/pam.h

```diff
@@ -0,0 +1,27 @@
+/************************************************************\
+ * Copyright 2022 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef HAVE_IMP_PAM_H_
+#define HAVE_IMP_PAM_H_ 1
+
+#include "imp_log.h"
+#include <security/pam_appl.h>
+#include <security/pam_misc.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#define FLUX_SERVICE_PAM "flux"
```

Comment:
This define should also be moved into `pam.c` I think, since it should be private to that file.